### DISCUSSION TITLE
OpenAPI spec integration with API playground in Nextra?

### DISCUSSION THREAD

**Discussion by Abdullah-2906** [shuding/nextra#3353] [Q&A]
Is it possible to integrate OpenAPI spec for API documentation in Nextra, along with an interactive API playground, allowing users to add an API key and make test requests directly from the docs?

**Comment by arno-fukuda** [comment]
I currently use [stoplight](https://github.com/stoplightio/elements):
https://docs.imgix.com/apis/management/api-playground

Alternatively, Fumadocs currently has a build in OpenAPI spec solution:
https://fumadocs.vercel.app/docs/ui/museum/buy-museum-tickets
